files/porter_bootstrap

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash -ex
+#!/bin/bash -e
```

Comment:
I don't know what the issue is. All the information i have is your commit message